### PR TITLE
Add name prop to fields containers divs

### PR DESCRIPTION
**Pull request by cesarlr**
hi @robinkeunen we have added name property to container divs to make inheritance easier

### CODE REVIEW

**Comment by cesarlr** [comment]
@remytms @robinkeunen What about this?

File: easy_my_coop_website/views/subscription_template.xml [hunk 1]
```diff
@@ -89,16 +89,20 @@
...
-                                    type="hidden"
-                                    name="csrf_token"
-                                    t-att-value="request.csrf_token()"
-                                />
-
-                                <div t-attf-class="form-group">
+                                <div name="crsf_token_container">
+                                    <p style="color:red;">
+                                        <t t-esc="error_msg" />
+                                    </p>
```
Comment (robinkeunen) [inline easy_my_coop_website/views/subscription_template.xml]: This banner is used to display error message that may be unrelated to csrf tokens. It should be out of the csrf div.

File: easy_my_coop_website/views/subscription_template.xml [hunk 2]
```diff
@@ -762,16 +800,23 @@
...
-                                    type="hidden"
-                                    name="csrf_token"
-                                    t-att-value="request.csrf_token()"
-                                />
 
-                                <div t-attf-class="form-group" style="display: none;">
+                                <div name="crsf_token_container">
+                                    <p style="color:red;">
+                                        <t t-esc="error_msg" />
+                                    </p>
```
Comment (robinkeunen) [inline easy_my_coop_website/views/subscription_template.xml]: same here.

**Review by robinkeunen — CHANGES_REQUESTED** [review]
Hi @cesarlr thanks for that long awaited dev request ;-)

Minor things to change for the csrf divs.

**Comment by cesarlr** [comment]
@robinkeunen Done!

**Review by enricostano — APPROVED** [review]

**Review by robinkeunen — APPROVED** [review]

**Review by remytms — APPROVED** [review]
:heart:

**Pull request merged**